src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -2624,7 +2624,51 @@ void ApplicationManagerImpl::UnregisterApplication(
     } else {
       resume_controller().RemoveApplicationFromSaved(app_to_remove);
     }
+
+    application_manager::VehicleInfoSubscriptions vehicle_data_for_unsubscribe =
```

Comment:
@IGapchuk could be const

Reply:
@AKalinich-Luxoft Fix in 11b04298dca9cf017301e7cf54e3b206cd43d883.